content/desktop/release-notes.md

```diff
@@ -345,7 +345,7 @@ For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/re
 
 ### Known Issues
 
-- Uninstalling Docker Desktop on Mac from the **Troubleshoot** page might trigger an unexpected fatal error popup.
+- Uninstalling Docker Desktop on Mac from the **Troubleshoot** page might trigger an unexpected fatal error po
```

Comment:
@aevesdocker I'm not sure if this was intentional 😅

Reply:
Woops! Fat fingers, will fix in a follow-up!

Reply:
omg I was so focused on the link I didn't even see this 🙈 

or maybe it's slang? 😆